tests/foreman/cli/test_repository.py

```diff
@@ -602,14 +602,16 @@ def test_positive_create_yum_repo_with_checksum_type(self):
         :CaseImportance: Critical
         """
         for checksum_type in u'sha1', u'sha256':
-            with self.subTest(checksum_type):
-                content_type = u'yum'
-                repository = self._make_repository({
-                    u'checksum-type': checksum_type,
-                    u'content-type': content_type,
-                })
-                self.assertEqual(repository['content-type'], content_type)
-                self.assertEqual(repository['checksum-type'], checksum_type)
+            for downloadpolicy in u'background', u'immediate':
+                with self.subTest(checksum_type):
+                    content_type = u'yum'
+                    repository = self._make_repository({
+                        u'checksum-type': checksum_type,
+                        u'content-type': content_type,
+                        u'download-policy': downloadpolicy,
+                        })
+                    self.assertEqual(repository['content-type'], content_type)
+                    self.assertEqual(repository['checksum-type'], checksum_type)
 
```

Comment:
assertion for download policy, if you think?

Reply:
I think not requires , because if something fails around download policy the repocreate would also fail too.
Another side we are also having tests around only download policy.